E7 Gear Optimizer/Hero.cs

```diff
@@ -57,11 +55,17 @@ public Hero(string ID, string name, List<Item> gear, Item artifact, int lvl, int
             {
                 string json = loadJson();
                 json = Encoding.UTF8.GetString(Encoding.Default.GetBytes(json)).Replace("✰", "");
-                json = json.Remove(json.IndexOf("\"skills\":")) + json.Substring(json.IndexOf("\"awakening\":"));
+                //json = json.Remove(json.IndexOf("\"skills\":")) + json.Substring(json.IndexOf("\"awakening\":"));
```

Comment:
This line was added because some users got an error when they added Tamarinne to the hero list. You can find the error message here https://pastebin.com/xYwWsgJi. I suspected it was caused by the ✰ character in the skill descriptions. Can you remove the description before parsing the string into json data?

Reply:
Unfortunately I cannot reproduce the error. Maybe we could add something like this:
```suggestion
if (json.Substring(0, 100).Contains("tamarinne"))//search only the beginning of file
{
    json = System.Text.RegularExpressions.Regex.Replace(json, "\"Shining Star.*\"", "\"Shining Star\"");
}

Reply:
I couldn't reproduce the error either. It only happens on a few systems.

"\\"Shining Star.*\\"" only matches if Shining Star is at the beginning of a string in the JSON. This does not work for the description of Song of the Forest and Serene Tune. 

Decreases cooldown of Shining Star✰ by 1 turn.\\"" would not be replaced for example. 

What do you think of this to remove the description of s1 and s2?
```suggestion  
if (json.Substring(0, 100).Contains("tamarinne"))//search only the beginning of file
{
    json = System.Text.RegularExpressions.Regex.Replace(json, "\"Shining Star.*\"", "\"Shining Star\"");
    json = System.Text.RegularExpressions.Regex.Replace(json, "\(Tamarinne\)[^\"]*", "");
}
```

Reply:
Yes, I totally agree. Skills' descriptions are not used by optimizer anyway.